source/html/renderer/src/card-elements.ts

```diff
@@ -1579,73 +1583,111 @@ class ActionCollection {
     private _owner: CardElement;
     private _actionButtons: Array<ActionButton> = [];
     private _actionCardContainer: HTMLDivElement;
-    private _expandedAction: ShowCardAction | HttpAction = null;
+    private _expandedAction: ShowCardAction = null;
     private _renderedActionCount: number = 0;
+    private _statusBar: HTMLElement = null;
+    private _actionCard: HTMLElement = null;
 
-    hideActionCardPane() {
-        this._actionCardContainer.innerHTML = '';
-        this._actionCardContainer.style.padding = "0px";
-        this._actionCardContainer.style.marginTop = "0px";
+    showStatus(action: Action) {
+        let renderedCard = action.status.render();
+        this._statusBar = renderedCard;
 
-        if (this.onHideActionCardPane) {
-            this.onHideActionCardPane();
-        }
+        this.refreshContainer();
+    }
 
-        if (this._expandedAction) {
-            raiseInlineCardExpandedEvent(this._expandedAction, false);
-        }
+    hideStatus() {
```

Comment:
Should be private and should be hideStatusCard for consistency with hideActionCard